pipelines/open-edx/mfe_build_and_deploy.yaml

```diff
@@ -0,0 +1,96 @@
+---
+resource_types:
+- name: rclone
+  type: docker-image
+  source:
+    repository: mitodl/concourse-rclone-resource
+    tag: latest
+
+resources:
+- name: mfe-app-(( mfe_path ))
+  type: git
+  source:
+    uri: (( mfe_repository ))
+    branch: (( open_edx_release_name ))
+- name: mfe-app-bucket
+  type: rclone
+  source:
+    config: |
+      [s3-remote]
+      type = s3
+      provider = AWS
+      env_auth = true
+      region = us-east-1
+
+jobs:
+- name: compile-and-deploy-mfe-(( mfe_path ))-to-(( open_edx_environment ))
+  plan:
+  - get: mfe-app-(( mfe_path ))
+    trigger: true
+  - task: compile-and-deploy-mfe
+    config:
+      platform: linux
+      image_resource:
+        type: registry-image
+        source:
+          repository: node
+          tag: (( mfe_node_version ))-bullseye-slim
+      inputs:
+      - name: mfe-app-(( mfe_path ))
+      outputs:
+      - name: mfe-app-(( mfe_path ))/dist
+      params:
+        ABOUT_US_URL:
+        ACCESS_TOKEN_COOKIE_NAME: (( open_edx_environment ))-edx-jwt-cookie-header-payload
+        BASE_URL: https://(( open_edx_lms_domain ))
+        CREDENTIALS_BASE_URL:
+        CSRF_TOKEN_API_PATH: /csrf/api/v1/token
+        Contact:
+        DISCOVERY_API_BASE_URL:
+        ECOMMERCE_BASE_URL:
+        FAVICON_URL: (( open_edx_favicon_url ))
+        HONOR_CODE_URL:
+        LANGUAGE_PREFERENCE_COOKIE_NAME:
+        LEARNING_BASE_URL:
+        LMS_BASE_URL: https://(( open_edx_lms_domain ))
+        LOGIN_URL: https://(( open_edx_lms_domain ))/login
+        LOGOUT_URL: https://(( open_edx_lms_domain ))/logout
+        LOGO_ALT_TEXT:
+        LOGO_TRADEMARK_URL: (( open_edx_logo_url ))
+        LOGO_URL: (( open_edx_logo_url ))
+        LOGO_WHITE_URL: (( open_edx_logo_url ))
+        MARKETING_SITE_BASE_URL: https://(( marketing_site_domain ))
+        ORDER_HISTORY_URL:
+        PRIVACY_POLICY_URL:
+        PUBLIC_PATH: /(( mfe_path ))/
+        REFRESH_ACCESS_TOKEN_ENDPOINT: https://(( open_edx_lms_domain ))/login_refresh
+        SEARCH_CATALOG_URL: https://(( open_edx_lms_domain ))/courses
+        SESSION_COOKIE_DOMAIN: (( open_edx_lms_domain ))
+        SHOW_FULLNAME:
+        SITE_NAME: (( open_edx_site_name ))
+        SITE_URL:
+        STUDIO_BASE_URL: https://(( open_edx_studio_domain ))
+        SUPPORT_CENTER_TEXT:
+        SUPPORT_CENTER_URL:
+        SUPPORT_URL: https://(( support_url ))
+        TERMS_OF_SERVICE_URL:
```

Comment:
xpro: https://xpro.mit.edu/terms-of-service/
mitxonline: https://mitxonline.mit.edu/terms-of-service/
residential: https://lms.mitx.mit.edu/tos